### ISSUE TITLE
iOS: message list doesn't respect system text size

### ISSUE THREAD

**Issue by gnprice** [zulip/zulip-mobile#2559]
On my iPhone 6 running iOS 10, I had the Zulip app open to the message list screen, then went over to system settings and changed the text size. The text in the app bar responded appropriately, but the message list didn't change.

This would make most of the app's functionality totally unusable for someone with a visual impairment who needs a large text size.

Compare #444, where a version of the app a year ago didn't respond to text size changes at all except on restart.

I haven't investigated exactly when this happens. If it is (as I assume) mainly about the message list, we're probably dropping the information about the text size when interacting with the webview.

**Comment by gnprice** [comment]
​Just to be clear: this is definitely not the appropriate behavior for our
app. The user doesn't know or care that we've used a webview to implement
the message list -- they just want the text size to be the way they've
configured it.

If you'd like to work up a fix (by making the text in the webview scale in
tandem with the rest of the app), that'd be great. I agree, this is an area
where changes should be tested on a variety of OS versions.

I just tested on Android (8.1 O, with 13.4.88). The text in the message
list scales with my system font size just like the rest of the text in the
app. (When I change the setting and come back to the app, it jumps back to
the unreads screen in the nav; that's slightly annoying but should rarely
matter for users other than ourselves.)

**Comment by gnprice** [comment]
Bumped the priority because we just heard (on support@) from a user who's running into this issue -- making messages unreadable.
